### PR TITLE
Square Pasture 2016 December Bronze

### PR DESCRIPTION
**Pull request by maggie-j-liu**
I added a C++ solution to Square Pasture (2016 dec bronze).

### CODE REVIEW

**Comment by tachithanhdanh** [comment]
Opps I also added one solution to this problem
https://github.com/cpinitiative/usaco-guide/pull/656

**Comment by tachithanhdanh** [comment]
Your implementation looks clearer than mine

**Comment by maggie-j-liu** [comment]
Sorry, I guess I missed that since I only looked at the titles of the pull requests, I'll look more closely next time.

File: solutions/USACO dec16-square-pasture.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    top = max(y2, y4);
+    int side = max(right - left, top - bottom);
+    cout << side * side << endl;
+    return 0;
+}
+```
+
+</CPPSection>
+
+</LanguageSection>
```
Comment (Farrius) [inline solutions/USACO dec16-square-pasture.mdx:41]: Looks good for me!

**Review by Farrius — APPROVED** [review]

**Comment by Farrius** [comment]
Ups, seems that there is another pull request in this solution. Some core-team member will decide what solution to include.

**Review by darren-yao — APPROVED** [review]
Approved

**Pull request merged**